src/site/content/en/blog/badging-api/index.md

```diff
@@ -158,9 +153,52 @@ Android only ever shows a dot instead of a numeric value.
 Don't assume anything about how the user agent wants to display the badge.
```

Comment:
**For the paragraph starting 'In some cases'.**

In some cases or on some OSs?

'In this case' is the start of a new sentence. It needs to change to 'In such cases'.

'while the Badging API': Do you mean 'because the Badging API'?